### ISSUE TITLE
It's a regression problem, right?

### ISSUE THREAD

**Issue by Sinha-Ujjawal** [MIT-AI-Accelerator/MagNav.jl#15]
It's a regression problem, right? If yes, how to find gradients (time derivative) of flux gate readings?

**Comment by Sinha-Ujjawal** [comment]
Also, the model will be trained once right?

**Comment by gnadt** [comment]
Yes, it's a regression problem. There's a flux gradient example in runs/sample.jl. Challenge problem participants determine the training setup for the model – judges only use the final model on the evaluation dataset.

**Comment by Sinha-Ujjawal** [comment]
Thx I saw the example, but the gradients require future flux values, which in real case scenarios will not be present, right? Tell me if I didn't understood the complete picture

**Comment by Sinha-Ujjawal** [comment]
The gradients are calculated using compute_fdm function, right?

**Comment by gnadt** [comment]
For the scope of this challenge problem it doesn't matter, but yes a backward scheme would be more appropriate for a real case scenario.
